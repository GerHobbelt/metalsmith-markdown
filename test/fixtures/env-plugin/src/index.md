---
title: This is my page title
---

Shall we render the @title now?
