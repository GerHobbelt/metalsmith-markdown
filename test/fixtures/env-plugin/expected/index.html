<p>Shall we render the This is my page title now?</p>
